src/Bundle/Controller/ResourceController.php

```diff
@@ -202,6 +234,10 @@ public function createAction(Request $request): Response
 
             $postEvent = $this->eventDispatcher->dispatchPostEvent(ResourceActions::CREATE, $configuration, $newResource);
 
+            if (null !== $output = $configuration->getOutput()) {
```

Comment:
@lchrusciel  Here, I do not use the resource factory (to create the Doctrine entity) and the data transformer (to transform into the DTO), cause we can directly use a null value on the form.

I'm not sure if we have to add a way to use a factory method for the DTO or sth else.

@pocky WDTY ?